openhands-cli/openhands_cli/simple_main.py

```diff
@@ -4,18 +4,27 @@
 This is a simplified version that demonstrates the TUI functionality.
 """
 
+import logging
+import os
+
+debug_env = os.getenv('DEBUG', 'false').lower()
+if debug_env != '1' and debug_env != 'true':
+    logging.disable(logging.CRITICAL)
+
 from prompt_toolkit import print_formatted_text
```

Comment:
Shall we still show warnings and error? We should have VERY small set of them in agent SDK. These could be useful to warn user about the configuration error